### ISSUE TITLE
Add ability to load multiply _ENV files

### ISSUE THREAD

**Issue by KarelWintersky** [vlucas/phpdotenv#393]
Pls, add ability to load multiply env files, smth like this:
```
$dotenv = \Dotenv\Dotenv::create(__DIR__, ['_env', '_env.db'] );
$dotenv->load();
```

P.S. it NOT a duplicate of #205 , it is a feature request

**Comment by GrahamCampbell** [comment]
Does:

```php
Dotenv\Dotenv::create(__DIR__, '_env')->load();
Dotenv\Dotenv::create(__DIR__, '_env.db')->load();
```

do what you need?

---

It's going to be hard to allow multiple files, in fact, a breaking change. Our current semantics is to stop as soon as we find the first matching file, if multiple directories are passed, for example. Passing multiple files, the only behaviour we could do would be to load the first match only.

**Comment by GrahamCampbell** [comment]
I'll have a think how this could be implemented in a way that makes sense, and doesn't break stuff.

**Comment by KarelWintersky** [comment]
Yeah, it works. 

I must make some experiments with values overriding and nesting values. 

Thank you!

**Comment by GrahamCampbell** [comment]
@KarelWintersky Please let me know what you think of https://github.com/vlucas/phpdotenv/pull/394. Does it do what you need?

You can install it using;

```
    "require": {
        "vlucas/phpdotenv": "dev-shortcircuit-config"
    },
```

Your usage requires passing a third parameter `false` to disable short circuit file loading:

```php
$dotenv = \Dotenv\Dotenv::create(__DIR__, ['_env', '_env.db'], false);
$dotenv->load();
```

If no third param is passed, or `true`, then we'll stop looking for files as soon as we find the first, which is the current behaviour, to avoid a breaking change.

**Comment by KarelWintersky** [comment]
> Does it do what you need?

Yes. Thank you very much.

It is difficult for me to imagine a situation in which I need to look for configuration files in two or more paths:
```
\Dotenv\Dotenv::create(['/var/www/1/', '/var/www/2/' ].... 
```
Why is this needed?

**Comment by GrahamCampbell** [comment]
> It is difficult for me to imagine a situation in which I need to look for configuration files in two or more paths:

Some people like phpdotenv to look in the parent directory for a `.env` file or in their `HOME` folder, as a fallback. After this PR, they could also use this functionality to merge in those files, instead of falling back to them.

**Comment by KarelWintersky** [comment]
Composer package will be updated later?

**Comment by GrahamCampbell** [comment]
This code is currently available as:

```
    "require": {
        "vlucas/phpdotenv": "^4.1@dev"
    },
```

**Comment by GrahamCampbell** [comment]
I'm going to give some more time for feedback before I release 4.1.0.

**Comment by GrahamCampbell** [comment]
> Is there a workaround available for this, besides calling the same functions twice with different paths? 

Yes, as I said already, both behaviours are implemented and are available, since 4.1.0.
